Update mm.py

LMFonville opened the pull request:
Modified (f)ALFF images to return z-scores rather than raw values. Raw output images are still in output dict but not used in calculations for csv files.  Added percentage of FD>0.5 outlier volumes to go along with simple count.

Commit:
Update mm.py

Modified (f)ALFF images to return z-scores rather than raw values. Raw output images are still in output dict but not used in calculations for csv files. 
Added percentage of FD>0.5 outlier volumes to go along with simple count.

stnava commented on antspymm/mm.py:
```diff
@@ -4011,6 +4013,7 @@ def resting_state_fmri_networks( fmri, fmri_template, t1, t1segmentation,
   outdict['ssnr'] = slice_snr( corrmo['motion_corrected'], csfAndWM, gmseg )
   outdict['dvars'] = dvars( corrmo['motion_corrected'], gmseg )
   outdict['high_motion_count'] = (rsfNuisance['FD'] > 0.5 ).sum()
+  outdict['high_motion_pct'] = (rsfNuisance['FD'] > 0.5 ).sum() / rsfNuisance.shape[0]
```
I don't think 0.5 is the same here as in Powers papers .... either (a) this needs to be a variable or (b) we need to adjust the FD calculation to best match ">0.5 is bad" standards ... I would guess that this threshold will lead to roughly half the volumes being labeled "bad" ... for this reason, I think this should be removed until the above issues are resolved.

LMFonville replied:
Agree there's no reason we should adopt "FD>0.5 means bad data". I think option a would be good to have so we can at least report on movement in the data in a way that matches other datasets (if I'm understanding the issue correctly). Choice of denoising will certainly impact outcomes..

stnava replied:
ok - that's fine - then pass 'FD_threshold' as a variable s.t. we can determine its appropriate setting later ...

stnava reviewed (commented):
reducing this value https://github.com/stnava/ANTsPyMM/blob/31198537bd28ecbaaf28f40da02f4b5a5f44c08c/antspymm/mm.py#L865

and also this value https://github.com/stnava/ANTsPyMM/blob/31198537bd28ecbaaf28f40da02f4b5a5f44c08c/antspymm/mm.py#L1119 

will change the range of the FD calculation ... maybe value 2 would do it?  I don't know if there is a "reference" FD out there against which we could compare.

stnava commented:
I have been thinking more about this - I would prefer to make this a minimal change ie keep the raw ALFF and fALFF images as they are .... however, you can also add "mean_alff" "mean_falff" and "sd_alff" and "sd_falff" to the data frame ( and output - note that these need to be passed to the output df too ) ..... so a user could post-hoc do this z-score transformation.  

if we can show that z-scoring is better  (later) - then we can make z-score the default.

LMFonville commented:
Yeah that sounds like a good alternative. Will make the necessary changes to the PR.

LMFonville committed:
Add files via upload

Added global mean and standard deviation for alff/falff and FD_threshold variable
